Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/common/exception/InvalidVaccinationCertificateException.kt

```diff
@@ -57,6 +34,7 @@ class InvalidVaccinationCertificateException(
         }
 }
 
-private const val ERROR_MESSAGE_VC_INVALID = R.string.error_vc_invalid
+private const val PREFIX = "VC_"
```

Comment:
is it for Vaccination certificates error codes or for all types of certificates? and is it required to add this prefix ?
I found very difficult to find the actual error code , if you look for `VC_PREFIX_INVALID` as the error states

Reply:
The prefix is part of the tech spec for vaccination - it is only affecting error messages specific to vaccination. The idea is that InvalidVaccinationCertificateException handles specific errors/ specific error messages for vaccination, while InvalidHealthCertificateException has general errors for all certificate types.